net/netbird/Makefile

```diff
@@ -40,6 +40,10 @@ define Package/netbird/description
   gateways, and so forth.
 endef
 
+define Package/netbird/conffiles
```

Comment:
This change looks like it is going to be needed in OpenWrt 23.05 branch. Are you planning to create backport PR once this is merged?